### DISCUSSION TITLE
is there any solution to get package info from cross origin

### DISCUSSION THREAD

**Discussion by henrygo2208** [npm/feedback#766] [Registry]
as the title, i want to get package info in browser

**Comment by MylesBorins** [comment]
This should be possible with the appropriate header

https://github.blog/changelog/2022-04-21-support-for-cors-response-headers/

**Reply by henrygo2208** [reply]
thanks, it is very helpful. is there more detailed specification about registry.npmjs.com endpoints? Like, how to search packages.

**Reply by MylesBorins** [reply]
We do not have official / up to date registry docs... but https://github.com/npm/registry has some historic details that still mostly work

**Reply by henrygo2208** [reply]
it's enough. thank you very much.
